lib/cryptoexchange/exchanges/bitcoin_indonesia/bitcoin_indonesia.yml

```diff
@@ -0,0 +1,19 @@
+:pairs:
```

Comment:
@tmlee not that it matters, but do we need the extra colon in front? less keystrokes bro....